rpcs3/rpcs3qt/tooltips.h

```diff
@@ -55,15 +55,15 @@ class Tooltips : public QObject
-		const QString spu_fast                  = tr("This is slower than the SPU Recompiler but significantly faster than the precise interpreter.\nGames rarely need this however.");
-		const QString spu_asmjit                = tr("This is the fast option with very good compatibility.\nIf unsure, use this option.");
-		const QString spu_llvm                  = tr("This is the fastest option with very good compatibility.\nRecompiles the game's SPU LLVM cache before running which adds extra start-up time.\nIf you experience issues, use the ASMJIT Recompiler.");
-		const QString accurate_xfloat           = tr("Fixes bugs in various games at the cost of performance.\nThis setting is only applied when SPU LLVM is active.");
-		const QString spu_cache                 = tr("Should normally stay enabled.\nDisable this if the cache becomes too large.\nDisabling it does not remove the existing cache.");
+		const QString ppu_precise               = tr("Interprets PPU code with absolute accuracy.\nThis is the most accurate Interpreter, but very slow to play games with.\nYou may try this as a last resort if you encounter odd bugs or crashes.\nIf unsure, use PPU Interpreter Fast or PPU Recompiler (LLVM).");
+		const QString ppu_fast                  = tr("Interprets PPU code with sacrificed accuracy in order to achieve better performance.\nThis is the fastest interpreter.\nIt very rarely breaks games even in comparison to the Precise option.\nTry this if PPU Recompiler (LLVM) fails.");
+		const QString ppu_llvm                  = tr("Recompiles and caches game's PPU code using LLVM Recompiler once before running it for the first time.\nThis is by far the fastest option and should always be used.\nShould you face compatibility issues, fall back to one of the Interpreters and retry.\nIf unsure, use this option.");
+		const QString spu_precise               = tr("Interprets SPU code with absolute accuracy.\nThis is extremely slow but may fix broken graphics in some games.");
+		const QString spu_fast                  = tr("Interprets SPU code with sacrificed accuracy in order to achieve better performance.\nThis is slower than the SPU Recompiler but significantly faster than the precise interpreter.\nGames rarely need this however.");
+		const QString spu_asmjit                = tr("Recompiles game's SPU code using ASMJIT Recompiler.\nThis is the fast option with very good compatibility.\nIf unsure, use this option.");
+		const QString spu_llvm                  = tr("Recompiles and caches game's SPU code using LLVM Recompiler before running which adds extra start-up time.\nThis is the fastest option with very good compatibility.\nIf you experience issues, use the ASMJIT Recompiler.");
+		const QString accurate_xfloat           = tr("Adds extra accuracy to float vectors processing.\nFixes bugs in various games at the cost of performance.\nThis setting is only applied when SPU LLVM is active.");
```

Comment:
Accurate xfloat can also be used on the Fast interpreter

Reply:
Well this piece was there before me. Should I remove this sentence or somehow change it?

Reply:
It probably was never updated. Should just change it to reflect that it can be used only on Fast and LLVM.

Reply:
Ok

Reply:
I remember some PRs ago some checkboxes were disabled in certain settings set. Maybe not change the tooltip but disable the checkbox at all when spu is not fast or llvm

Reply:
The checkbox is already greyed out when other options are selected

Reply:
Done